nofos/nofos/templates/nofos/nofo_edit_base.html

```diff
@@ -0,0 +1,79 @@
+{% extends 'base.html' %}
```

Comment:
I think I had a slightly different vision for what this could be.

I was thinking we could keep the edit templates more or less as they were, but then create a macro here for the table-building part of the HTML, so everything between the `{% if subsection_matches %}` block. We could also stick the JS code in a script that is added at the end of the table so that it is only included where that table needs to be built. 

Similar issue with the Form classes above: the thing that is different about these pages (compared to our other edit_{attribute_name} templates) is that we are building a table of subsections, so I would much prefer if just the table was sheared off into its own macro.

What do you think, am I missing something?